FirebaseSessions/Sources/ApplicationInfo.swift

```diff
@@ -63,4 +66,15 @@ class ApplicationInfo: ApplicationInfoProtocol {
   var mccMNC: String {
     return FIRSESValidateMccMnc(networkInfo.mobileCountryCode, networkInfo.mobileNetworkCode) ?? ""
   }
+
+  var environment: String {
+    if let env = ProcessInfo.processInfo.environment["FIREBASE_RUN_ENVIRONMENT"] {
```

Comment:
`ProcessInfo.processInfo.environment` are environment variables passed to the app. `ProcessInfo.processInfo.arguments` is how we get command line arguments. I think it'd be good for us to use command line arguments instead for consistency.

Reply:
I don't think arguments is a great fit here. Arguments is an array of Strings, whereas Environment is a map of String to String. With a MAP, it is easy to get a key-value pair. With an array it is usually beneficial only for the availability or unavailability of a flag.  Our usecase needs finding the value rather than just checking availability of the value.

https://developer.apple.com/documentation/foundation/processinfo/1417911-environment
https://developer.apple.com/documentation/foundation/processinfo/1415596-arguments